### ISSUE TITLE
Power Control Panel on the Ayn Loki

### ISSUE THREAD

**Issue by AMVONE** [project-sbc/Power-Control-Panel-v2#58]
This is not an issue but more of a question, is there any thought about this program being available on the Loki range of handhelds once they come out?

**Comment by project-sbc** [comment]
Yes it will be compatible, I have a big update in the works that will really make it more usabel

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: AMVONE ***@***.***>
Sent: Monday, December 12, 2022 6:14:49 PM
To: project-sbc/Power-Control-Panel-v2 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [project-sbc/Power-Control-Panel-v2] Power Control Panel on the Ayn Loki (Issue #58)


This is not an issue but more of a question, is there any thought about this program being available on the Loki range of handhelds once they come out?
